buku.py

```diff
@@ -2454,7 +2561,14 @@ def main():
         if len(keywords) > 1:
             tags = parse_tags(keywords[1:])
 
-        bdb.add_rec(args.add[0], title_in, tags, desc_in, args.immutable)
+        url = args.add[0]
```

Comment:
Can we also ensure `buku --write` opens the editor (even if `-a` is not passed) and adds a bookmark if added from editor?